stories/utils/generate-widget-stories.js

```diff
@@ -210,24 +212,27 @@ export function generateReportBasedWidgetStories( {
 		...customVariants,
 	};
 
-	let widget;
+	const slugs = moduleSlugs.map( ( slug ) => {
+		return `${ slug }-widget`;
+	} );
+	const slug = slugs.join( ' ' );
```

Comment:
Let's use shorter version:

```suggestion
	const slug = moduleSlugs.map( ( slug ) => `${ slug }-widget` ).join( ' ' );
```